target/zephyr/Kconfig

```diff
@@ -387,6 +387,13 @@ if MENDER_MCU_CLIENT
 
         menu "Error detection and recovery"
 
+            config MENDER_MAX_STATE_DATA_STORE_COUNT
+                int "Max state data store count"
```

Comment:
This doesn't really reflect what the `help` says and can be quite confusing. I think keeping the name of the macro is fine, but this description needs a better wording.